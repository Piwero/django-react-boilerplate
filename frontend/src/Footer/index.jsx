import React from "react";
const Footer=() =>{
    return(
        <div className = "flex place-content-center py-4">
            <p>Piwero</p>
        </div>
    )
}
export default Footer