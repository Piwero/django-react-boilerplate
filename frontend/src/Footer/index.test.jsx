import React from "react"
import renderer from 'react-test-renderer';
import Footer from ".";

describe(`Footer`, () => {
  it(`renders correctly`, () => {
    const footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot()
  })
})

