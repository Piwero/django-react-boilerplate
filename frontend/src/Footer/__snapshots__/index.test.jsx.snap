// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="flex place-content-center py-4"
>
  <p>
    Piwero
  </p>
</div>
`;
